package.json

```diff
@@ -33,6 +33,7 @@
     "bip44-constants": "^128.0.0",
     "ethereumjs-util": "^7.1.3",
     "react": "^18.1.0",
+    "secp256k1": "^5.0.1",
```

Comment:
Since this package is used in ambire-common (src/libs/keystoreSigner/keystoreSigner.ts), shouldn't we put this package as a "dependency" here? And then - the extension should not have this package installed separately?